### PR TITLE
Add user stats from db to stats page

### LINKED ISSUE #154: Implement fetching user stats from db on stats page

### PR DESCRIPTION
**Pull request by Rebecca-CYLi**
fixes #154 

Takes user statistic gameScore in db and puts it to see in stats.html

### CODE REVIEW

File: flask/app/rps.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ def play():
     return render_template("play.html")
 
 
-@rps.route("/stats")
-def stats():
-    return render_template("stats.html")
+# @rps.route("/stats")
```
Comment (calvh) [inline flask/app/rps.py:24]: remove commented lines

File: flask/app/rps.py [hunk 2]
```diff
@@ -38,7 +38,10 @@ def scores():
         db_result = Users.find_one({"_id": ObjectId(id)}, {"gameScore": 1})
 
         if db_result:
-            return dumps(db_result["gameScore"]), 200
+            # db_dump = dumps(db_result["gameScore"])
```
Comment (calvh) [inline flask/app/rps.py:41]: remove commented lines

File: flask/app/templates/stats.html [hunk 3]
```diff
@@ -9,9 +9,9 @@ <h1 style="background-color:transparent; margin-top: -175px; padding: 0px;">Play
         <img src="./static/img/female.png" alt="Avatar1" style="border-radius:50%; height: 300px; width: 300px;">
         <h2>You</h2>
         <div style="text-align: left; margin-left: 20%;">
-          <p>Wins: #</p>
-          <p>Losses: #</p>
-          <p>Ties: #</p>
+          <p>Wins: <span id="my-wins"></span></p>
```
Comment (calvh) [inline flask/app/templates/stats.html:12]: I have been using "player" all over, so lets keep it consistent and use "player-wins" etc.

File: flask/app/rps.py [hunk 4]
```diff
@@ -38,7 +38,10 @@ def scores():
...
+            json_db = loads(dumps(db_result["gameScore"]))
+            return render_template("stats.html", json_db=json_db)
```
Comment (calvh) [inline flask/app/rps.py:43]: this needs to be in a new route for stats

**Review by calvh — CHANGES_REQUESTED** [review]
Sorry, the code doesn't make sense. You are defining a render method for the "stats" page on the "scores" route. The scores route is an API route, not a render route.

Option 1
1) Revert all the changes to the "scores" route
2) Define a separate route for "stats"
3) In your stats route, do your db query and get the data, then inject it into the stats template, which you will render.
4) In the stats jinja template, define the appropriate blocks to receive and display the data injected by the route you defined.

Option 2
Alternatively, add JS code to the stats page to fetch the data from the scores API and modify the DOM appropriately.

I suggest Option 1 since we are using jinja templates

File: flask/app/rps.py (hunk 1 above)
Comment (Rebecca-CYLi) [reply flask/app/rps.py]: done

File: flask/app/rps.py (hunk 2 above)
Comment (Rebecca-CYLi) [reply flask/app/rps.py]: done

File: flask/app/rps.py (hunk 4 above)
Comment (Rebecca-CYLi) [reply flask/app/rps.py]: done

File: flask/app/templates/stats.html (hunk 3 above)
Comment (Rebecca-CYLi) [reply flask/app/templates/stats.html]: fixed

File: flask/app/rps.py (hunk 4 above)
Comment (Rebecca-CYLi) [reply flask/app/rps.py]: fixed

**Review by calvh — APPROVED** [review]

**Pull request merged**